superbit_lensing/shear_profiles/shear_plots.py

```diff
@@ -2,7 +2,7 @@
 from matplotlib import rc,rcParams
 import matplotlib.pyplot as plt
 from astropy.table import Table
-import pudb
+import pudb,pdb
 
 def compute_alpha(nfw, radius, gtan, variance):
```

Comment:
We shouldn't' have alpha be computed in two different places in the code. Now that you are competing it in `annular_jmac.py` and adding it to the table metadata, we should remove this